docs/.vitepress/config.ts

```diff
@@ -91,6 +91,7 @@ export default defineConfig({
     zh: { label: '简体中文', link: 'https://cn.vitejs.dev' },
     ja: { label: '日本語', link: 'https://ja.vitejs.dev' },
     es: { label: 'Español', link: 'https://es.vitejs.dev' },
+    pt: { label: 'Português', link: 'https://pt.vitejs.dev' },
```

Comment:
This site doesn't actually exist.
```suggestion
```

Reply:
@tony19 the source of the site is ready, it is waiting to be served
by the people from vite team. I already did the pull request to
inform the conclusion for the first version of the translated source.

So could you at least wait until the site is served? :+1:

Reply:
Ok. When that happens, the change will automatically propagate to this repo (via a GitHub bot that merges changes from the main `vite` repo), so there's no need to add the locale explicitly here.

Reply:
Good, I did not know about that :smile: .
@tony19 So can you only pick the others updates except this?

Reply:
Yes, I prefer to keep the docs in sync with the upstream repo.